Assets/Code/SceneManager.cs

```diff
@@ -24,14 +24,19 @@ void Main ()
         Debug.Log ("Main");
     }
 
+    private void InitializeScene()
+    {
+        PositionGrid grid = GameObject.FindWithTag("SceneManager").GetComponent(typeof(PositionGrid)) as PositionGrid;
+        grid.AssignTransformPositionsToTiles();
+        Instantiate(scenePlayer, charPosition, Quaternion.identity);
+        Instantiate(sceneEnemy, enemyPosition, Quaternion.identity);
+        grid.AssignOwnershipOfTiles();
```

Comment:
I think after we have changed how the tiles are assigned to a team, we can move the `grid.AssignTransformPositionsToTiles();` and `grid.AssignOwnershipOfTiles();` back into `PositionGrid`'s initialization function.